src/realm/array_blob.cpp

```diff
@@ -64,39 +64,49 @@ BinaryData ArrayBlob::get_at(size_t& pos) const noexcept
 
 ref_type ArrayBlob::replace(size_t begin, size_t end, const char* data, size_t data_size, bool add_zero_term)
 {
+    size_t sz = blob_size();
```

Comment:
Looks like `blob_size()` is only defined in debug mode, it's why CI is failing. I'm not sure what the reason for that is though, I don't mind if we move it to production.